gui/app/directives/modals/viewers/viewerDetailsModal.js

```diff
@@ -376,6 +383,14 @@
                     });
                 };
 
+                $ctrl.disableActiveUserList = () => {
```

Comment:
Update this to `disableActiveUserListChange` to keep in line with the naming convention of the other similar function `disableAutoStatAccuralChange `